docs/timeseries/index.rst

```diff
@@ -39,19 +41,22 @@ In this section, we take a quick look at how to read in a time series, access
 the data, and carry out some basic analysis. For more details about creating and
 using time series, see the full documentation in :ref:`using-timeseries`.
 
-The simplest time series class is |TimeSeries| - it represents a time series as
-a collection of values at specific points in time. If you are interested in
+The most minimal time series class is |TimeSeries| — it represents a time series
```

Comment:
"most basic"? Maybe it's just me with minimal and timeseries 🤷

Reply:
Most basic works!